AncientReligions/events/ACR_hellenic_revival_events.txt

```diff
@@ -189,7 +197,12 @@ character_event = {
 			limit = { religion_group = jewish_group }
 			add_trait = sympathy_judaism
 		}
-		religion = hellenic_pagan
+		random_list = {
+			75 = { religion = hellenic_pagan }
```

Comment:
I almost suggested making the heresies rarer, but then thought better of it.
With this distribution, the players have a decent-but-small chance of picking one of the heresies if that is what they are after.